### DISCUSSION TITLE
Cannot access behind Traefik

### DISCUSSION THREAD

**Discussion by pathwayx99** [doitandbedone/ispyagentdvr-docker#307] [Q&A]
Hi everyone. 

I've been running this image behind a Traefik proxy for some time now without issue, along with several other images, most of them with a web interface. It's been working fine without any problems whatsoever.

However, recently, when trying to access the UI, I get a blank page using Firefox. Edge gives me HTTP error 406. When I remove Traefik and access the UI direct by IP address, it works fine. When I put Traefik back up again, same thing. The weird thing is that Traefik shows everything green. Entrypoint, router, middleware, service. The certificate is fine. The name resolution is fine. All other web interfaces with identical setup work fine.

So, since I'm at my wits' end, I wanted to ask if anyone knows of a breaking change (I'm running Watchtower, so containers update automatically) that could cause the Agent UI not to be accessible behind a Traefik proxy?

Thanks in advance for any input!

**Comment by doitandbedone** [comment]
Sorry for the very late reply. Perhaps it has to do with ports. The image requires several ports for it to work properly. Check this out: https://stackoverflow.com/questions/59830648/traefik-multiple-port-bindings-for-the-same-host-v2
https://community.traefik.io/t/docker-container-with-multiple-ports/4657

**Comment by AMD-NICK** [comment]
Look at this https://github.com/doitandbedone/ispyagentdvr-docker/issues/357#issuecomment-1206978330

**Reply by pathwayx99** [reply]
Hey, thanks. I've reached the same point, actually. I can use Traefik, but no https. Which is a bummer.

What gets me is that it _used_ to work...
